### ISSUE TITLE
Stripping whitespace from translated text in dialog mode

### ISSUE THREAD

**Issue by jotumn** [SethRobinson/UGT#35]
For example, this image:
![Image1](https://user-images.githubusercontent.com/16961866/128596240-c3b7b707-2b74-452a-9ca1-ef68d0d7b69f.png)

Produces the following text in the copy buffer:
```

I have astral vision and I'm on the lookout for wizards.


```
With one empty line at start and two in the end.

The log strips it out correctly, btw:
```
あたしはアストラル視覚で魔法使いを警戒してるわ
I have astral vision and I'm on the lookout for wizards.
```

Also, the auto-download of update worked fine for me.

**Comment by SethRobinson** [comment]
Oops, will check that.

**Comment by SethRobinson** [comment]
Fixed in d16801e7  (for the clipboard, I even left off the final CR at the end of the line.  let me know if that doesn't work with what you're doing)
